packages/template-retail-react-app/app/components/promo-code/index.jsx

```diff
@@ -118,6 +118,7 @@ export const PromoCode = ({form, submitPromoCode, itemProps}) => {
                             justifyContent="flex-start"
                             variant="link"
                             fontSize="sm"
+                            color="blue.700"
```

Comment:
Axe tool is complaining about color contrast here, I bumped it up one shade from blue.600 to blue.700